src/XliffTasks/Model/XamlRuleDocument.cs

```diff
@@ -65,9 +65,9 @@ private static string GenerateIdForPropertyMetadata(XElement element)
 
         private static string? GetComment(XElement element, string attributeName)
```

Comment:
Nullables get complicated when moving from 3.1 to 6. Did you have any weird messages or errors like that?

Reply:
I think the "nullables" feature is something you have to explicitly turn on in 6 and I opted not to. In any case, no weird messages or errors.